addons/library.kodi.guilib/libKODI_guilib.h

```diff
@@ -825,13 +825,13 @@ class CAddonGUIRenderingControl
   virtual ~CAddonGUIRenderingControl();
   virtual void Init();
 
-  virtual bool Create(int x, int y, int w, int h, void *device);
+  virtual bool Create(int x, int y, int w, int h, float ratio);
```

Comment:
you mean VIZ_PROPS? that does not help. device is the DX9 rendering device which is required by an addon to render in the control.